### PR TITLE
feat(composition): best of technozaure event

### PR DESCRIPTION
**Pull request by Boyadjie**
## 🤔 Why?

To announce the event

## 💻 How?

By using the components, i've created the video under the folder `src/BestOfTechnozaure/`. And added the default configuration to the file `src/root.tsx`

## ✅ How to validate this PR?

- You can run `pnpm start` and go to localhost:3000. You should see the different compositions under the folder `Event`, the one named `BotzEvent` is the final composition

### CODE REVIEW

File: src/BestOfTechnozaure/Event/Contact.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			</Sequence>
+			{contact.mail && contact.phone && (
+				<Sequence name="Contact informations" from={10}>
+					<SlideTop from={830} to={930} durationInFrames={30}>
+						<FadeIn durationInFrames={30}>
+							<DoubleText firstText={contact.mail} secondText={contact.phone} />
+						</FadeIn>
+					</SlideTop>
+				</Sequence>
+			)}
```
Comment (Slocaly) [inline src/BestOfTechnozaure/Event/Contact.tsx:57]: Maybe the mail and the phone must be un-linked
With this code if one of them is not given, nothing appear

Comment (Boyadjie) [reply src/BestOfTechnozaure/Event/Contact.tsx:57]: Done

File: src/BestOfTechnozaure/Event/Categories.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+				return (
+					<div
+						style={{
+							position: 'relative',
+							display: 'flex',
+							justifyContent: 'center',
+							height: 275,
+						}}
+					>
```
[the remark this answers is not in the document]
Comment (Boyadjie) [reply src/BestOfTechnozaure/Event/Categories.tsx:39]: Done

File: src/BestOfTechnozaure/Event/Tribes.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+export const Tribes: React.FC<{
+	categories: CategoryProps[];
+}> = ({categories}) => {
+	return (
+		<AbsoluteFill>
+			<Sequence name="Title Categories">
+				<FadeIn durationInFrames={20}>
+					<FadeOut startingFrame={70} durationInFrames={15}>
+						<Title
+							title="VENEZ ASSISTER À DES CONFÉRENCES TECH"
```
Comment (CruuzAzul) [inline src/BestOfTechnozaure/Event/Tribes.tsx:18]: You already have `textTransform: 'uppercase'` in style 😊

**Review by CruuzAzul — APPROVED** [review]

File: src/BestOfTechnozaure/Event/Intro.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		<AbsoluteFill>
+			<Sequence name="Website">
+				<Footer content="www.zenika.com" />
+			</Sequence>
+			<Sequence name="Logo">
+				<Logo src="/BestOfTz/BOTZ_LOGO.png" width={sizeDown} top={60} />
+			</Sequence>
+			<Sequence name="Title" from={10}>
+				<FadeIn durationInFrames={20}>
+					<FadeOut startingFrame={100} durationInFrames={20}>
```
Comment (Boyadjie) [inline src/BestOfTechnozaure/Event/Intro.tsx:47]: Fade out not at starting at the same time as others

**Pull request merged**